makefile

```diff
@@ -81,6 +81,13 @@ test/triton:
 test/triton/dev:
 	./test/triton.sh
 
+
+# ------------------------------------------------
+# Multi-datacenter usage
+clean/triton:
```

Comment:
This line isn't an issue but `make build` is broken due to file path issue in `test/Dockerfile`. We'll also want to make sure `make test` still works. Local tests passed for me when I patched.